x/group/go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/regen-network/regen-ledger/x/group
+
+go 1.15
+
+require (
+	github.com/cockroachdb/apd/v2 v2.0.2
+	github.com/cosmos/cosmos-sdk v0.42.4
+	github.com/gogo/protobuf v1.3.3
+	github.com/gorilla/mux v1.8.0
+	github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	github.com/regen-network/cosmos-proto v0.3.1
+	github.com/regen-network/regen-ledger v1.0.0
```

Comment:
Why does x/group depend on `github.com/regen-network/regen-ledger`? Doesn't that create a circular dependency?

Reply:
So `x/group` imports `"github.com/regen-network/regen-ledger/math"`. To avoid the go module `github.com/regen-network/regen-ledger/x/group` depending on the go module `github.com/regen-network/regen-ledger`, should we move `math` to the `types` go module? cc @aaronc

Reply:
Yes sure